tests/src/asn1_helpers.c

```diff
@@ -15,6 +15,8 @@
 
 #include <mbedtls/asn1.h>
 
+#include <test/asn1_helpers.h>
```

Comment:
Also, for a change of this magnitude, I think we need a 2.28 backport as well, just for the parts that backport cleanly. I don't particularly care to pass `-Wmissing-prototypes` in 2.28, but I want backporting changes to tests to remain easy when it's currently a simple cherry-pick. For parts that are already different, it doesn't matter.

Reply:
Who shall do the backporting?

Reply:
Ideally you, after the main pull request is approved. But if we've exhausted your bandwidth someone else will deal with it, it just might take more time.

Reply:
Ok, I'm happy to do it.